src/flags/salesforceId.ts

```diff
@@ -12,9 +12,9 @@ const messages = Messages.loadMessages('@salesforce/sf-plugins-core', 'messages'
 
 export type IdFlagConfig = {
   /**
-   * Can specify if the version must be 15 or 18 characters long.  Leave blank to allow either 15 or 18.
+   * Can specify if the version must be 15 or 18 characters long or 'both'. Leave blank to allow either 15 or 18.
    */
-  length?: 15 | 18;
+  length?: 15 | 18 | 'both';
```

Comment:
Wondering if this should be:

```typescript
length?: Many<15 | 18>
```

That would give us some flexibility in the scenario (albeit, highly unlikely) that another id length is added, in which case `both` no longer makes sense

Reply:
When @mshanemc and I discussed we were concerned about having to do:

length: 15
or
length 18
or 
[15,18]

with the array feeling kind of out of place, as a length, which is why 'both' was settled on. Shane did suggest 'all', but that felt a little to loose.